From 22412e651125bbec2200ff37055e5f739b57b9ff Mon Sep 17 00:00:00 2001
From: Gabriel Przybysz Gonçalves Júnior <gabriel.przybsyzgoncalvesjunior@windriver.com>
Date: Fri, 30 Oct 2025 11:52:01 -0300
Subject: [PATCH] csm-replication: add default values for templates

This patch consists of adding the default values in the values.yaml of
the helm chart.
It's necessary because templates need default values to be able
to build the package.

Signed-off-by: Gabriel Przybysz Gonçalves Júnior <gabriel.przybsyzgoncalvesjunior@windriver.com>
---
 charts/csm-replication/values.yaml | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/charts/csm-replication/values.yaml b/charts/csm-replication/values.yaml
index 9061208..8ca02de 100644
--- a/charts/csm-replication/values.yaml
+++ b/charts/csm-replication/values.yaml
@@ -47,3 +47,12 @@ allowPvcCreationOnTarget: "false"
 #  - ip: "10.10.10.11"
 #    hostnames:
 #      - "foo.baz"
+
+config:
+  clusterId: ""
+
+replicationGroup:
+  create: false
+
+kubectl:
+  image: registry.k8s.io/kubectl:v1.35.3
--
2.34.1

